frontend/app/components/input-radios.tsx

```diff
@@ -17,9 +17,10 @@ export interface InputRadiosProps {
   legend: ReactNode;
   name: string;
   required?: boolean;
+  children?: ReactNode;
```

Comment:
Remove this new prop and use `helpMessagePrimary`. You can use `helpMessagePrimaryClassName` to override the default styling.